operations/deployment-engineering/frequently-asked-questions.md

```diff
@@ -0,0 +1,66 @@
+# Deployment Engineering: Frequently Asked Questions
+
+## How is this team measured?
+
+Primary measurement is **Customer Seats Deployed** - % of customers with >$50K ARR with at least 80% of seats deployed.
+
+## How will work be allocated to this team?
+
+Deployment Engineering will work closely with other post-sales teams (support, customer success, TAMs) to determine the highest impact technical customer priorities for the team to work on.
+
+For instance, when choosing between three options, work will always be prioritized based on the impact level, with effort taken into consideration (e.g. a medium impact low effort technical priority may supersede a high impact but very high effort priority).
+
+Moreover, for any one-off customer solutions, a clear SOW or other such document will be created to set clear expectations with the customer of what is delivered, by whom, and in what timeframe, [outlined as a services request by the CSMs/TAMs](https://docs.google.com/document/d/1EbP4Ab7N2hEAv--DRCcZHk0XtlG1MFP8F8F0viaTM7I/edit), signed off by the customer. This process is similar to what would be done with a partner. The sign-off is not necessarily a formal contractual agreement, but at minimum a verbal agreement to ensure expectations are set clearly before work begins.
+
+## Who owns long-term maintenance of solutions built by this team?
+
+Most solutions should not require long-term maintenance, as they will be standalone scripts, plugins or other code that is not dependent on Mattermost server versions.
+
+For solutions that require maintenance, we will discuss ownership between Deployment Engineering, TAMs, R&D and a vendor on a case-by-case basis.
+
+## How does this role differ from TAMs (Technical Account Managers)?
+
+TAMs are responsible for ongoing relationship management with technical champions, and own pre-defined customer onboarding in standard deployment environments (ie. follow our gold standard deployments).
+
+On the other hand, Deployment Engineering supports non-standard deployments with complex technical problems, as well as work closely with customers on temporary one-off custom solutions & integrations based on [requirements outlined by the CSMs/TAMs](https://docs.google.com/document/d/1EbP4Ab7N2hEAv--DRCcZHk0XtlG1MFP8F8F0viaTM7I/edit).
+
+Thus, Deployment Engineering acts as an extension of R&D supporting post-sales and customer success on the most complex, technical problems, with expertise on how to run and deploy Mattermost and the platform as a whole.
+
+Finally, we encourage a representative from TAM to join weekly Deployment Engineering meetings, and vice versa, to help deepen collaboration & understanding of each team’s workstreams.
```

Comment:
should this be on the handbook? maybe it can be rephrased into what's expected instead of a call to action.

Reply:
Thanks for the feedback! Removing for now